File: programacao/c/respostas/operacoes/16.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int numero, inverso;
    int unidade, dezena, centena;

    printf("Qual número, entre 0 e 999? ");
    scanf("%d", &numero);
    
    unidade = numero % 10;
    dezena = (numero / 10) % 10;
    centena = (numero / 100) % 10;
    
    inverso = unidade * 100 + dezena * 10 + centena;

    printf("O número invertido eh %d\n", inverso);

    return 0;
}

File: programacao/python/respostas/operacoes/16.py
numero = int(input("Qual numero, entre 0 e 999? "))

unidade = numero % 10
dezena = (numero // 10) % 10
centena = (numero // 100) % 10

inverso = unidade * 100 + dezena * 10 + centena

print(f"O numero invertido eh {inverso}")
